arangod/VocBase/VocbaseInfo.cpp

```diff
@@ -232,10 +226,10 @@ Result CreateDatabaseInfo::extractUsers(VPackSlice const& users) {
   return Result();
 }
 
-Result CreateDatabaseInfo::extractOptions(VPackSlice const& options,
-                                          bool extractId, bool extractName) {
+Result CreateDatabaseInfo::extractOptions(VPackSlice options, bool extractId,
+                                          bool extractName) {
   if (options.isNone() || options.isNull()) {
-    return Result();
+    options = VPackSlice::emptyObjectSlice();
```

Comment:
why did it change?

Reply:
This is exactly the bugfix. When no extra data was sent along with the "create database" request, we exited early, and then didn't set any defaults correctly (e.g. replicationFactor was then hard-coded to 1 and not to a sensible minimum value >= `--cluster.min-replication-factor`).

Reply:
Thanks